src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {
+  useState,
+  ReactElement,
+  FC,
+  SyntheticEvent,
+  Children,
+  cloneElement,
+} from 'react';
+
+interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  children?: any | null;
```

Comment:
There's a ReactNode type, I think. I might have the name wrong.

Reply:
Yeah, I was trying that but ran into other TS issues I could not resolve. After I convert children arg to the ExpansionPanel I'll try again. Thanks.

Reply:
ReactNode was right...then had to cast to React.Element array to successfully iterate over it.